src/main/telemetry/hott.c

```diff
@@ -324,78 +330,19 @@ static void workAroundForHottTelemetryOnUsart(serialPort_t *instance, portMode_e
 {
     closeSerialPort(hottPort);
 
-    portOptions_e portOptions = telemetryConfig()->telemetry_inverted ? SERIAL_INVERTED : SERIAL_NOT_INVERTED;
-
-    if (telemetryConfig()->halfDuplex) {
-        portOptions |= SERIAL_BIDIR;
-    }
-
-    hottPort = openSerialPort(instance->identifier, FUNCTION_TELEMETRY_HOTT, NULL, HOTT_BAUDRATE, mode, portOptions);
```

Comment:
why has the SERIAL_BIDIR flag been removed?  I'm pretty sure this will break HoTT on F3/F7 targets using hardware ports?